contentcuration/contentcuration/views.py

```diff
@@ -53,11 +54,11 @@ def channel(request, channel_id):
     channel = get_object_or_404(Channel, id=channel_id, deleted=False)
     channel_serializer =  ChannelSerializer(channel)
 
-    accessible_channel_list = Channel.objects.filter( Q(deleted=False, public=True) | Q(deleted=False, editors__email__contains= request.user))
+    accessible_channel_list = Channel.objects.filter( Q(deleted=False, public=True) | Q(deleted=False, editors__email__contains= request.user.email))
     accessible_channel_list = ChannelListSerializer.setup_eager_loading(accessible_channel_list)
     accessible_channel_list_serializer = ChannelListSerializer(accessible_channel_list, many=True)
 
-    channel_list = accessible_channel_list.filter(editors__email__contains= request.user).exclude(id=channel_id).values("id", "name")
+    channel_list = accessible_channel_list.filter(editors__email__contains= request.user.email).exclude(id=channel_id).values("id", "name")
```

Comment:
I think I made this comment on another PR: shouldn't be `editors__email__contains`, as that means it'll match any editor whose *email address* contains the current user's address (as a substring), which is a security vulnerability (da@sfu.ca could edit channels belonging to qda@sfu.ca). It should work with just `editors__email=request.user.email`, or does it not work as just ``editors=request.user`` or `editors__contains=request.user`?